src/UserGuide/Master/stage/Delete-Data/TTL.md

```diff
@@ -21,29 +21,42 @@
 
 # TTL
 
-IoTDB supports storage-level TTL settings, which means it is able to delete old data automatically and periodically. The benefit of using TTL is that hopefully you can control the total disk space usage and prevent the machine from running out of disks. Moreover, the query performance may downgrade as the total number of files goes up and the memory usage also increase as there are more files. Timely removing such files helps to keep at a high query performance level and reduce memory usage.
+IoTDB supports device-level TTL settings, which means it is able to delete old data automatically and periodically. The benefit of using TTL is that hopefully you can control the total disk space usage and prevent the machine from running out of disks. Moreover, the query performance may downgrade as the total number of files goes up and the memory usage also increase as there are more files. Timely removing such files helps to keep at a high query performance level and reduce memory usage.
 
 The default unit of TTL is milliseconds. If the time precision in the configuration file changes to another, the TTL is still set to milliseconds.
 
-## Set TTL
-
-The SQL Statement for setting TTL is as follow:
-
-```
-IoTDB> set ttl to root.ln 3600000
-```
+When setting TTL, the system will look for all devices included in the set path and set TTL time for these devices. The system will delete expired data at the device granularity.
+After the device data expires, it will not be queryable, but the data in the disk file cannot be guaranteed to be deleted immediately (it will be deleted within a certain time), but it can be guaranteed to be deleted eventually.
+Considering the operational cost, the system will not immediately physically delete data exceeding TTL, but will delay physical deletion through merging. Therefore, before the data is physically deleted, if the TTL is reduced or lifted, it may cause data that was previously invisible due to TTL to reappear.
```

Comment:
This sentence says similar things to the previous sentence, may merge them.
Possible revision:

After the device data expires, it will not be queryable.
However,  due to operational costs, the expired data will not be physically deleted right after expiring.
The physical deletion is delayed until compaction.
Therefore...